### ISSUE TITLE
Validator number rule is not defined

### ISSUE THREAD

**Issue by moar55** [adonisjs/adonis-validation-provider#34]
Hi there. I am trying to create rules for a request using validator, however when i add the ‘number’ rule which is available on  [indicative documentation](http://indicative.adonisjs.com/docs/number); I get this error:

```JSON
[
    {
        "field": "cost",
        "validation": "ENGINE_EXCEPTION",
        "message": "number is not defined as a validation"
    }
]
```
here is my code:

```javascript
  const rules = {
  manufacturer: 'required|integer',
  cost: 'required|number',
  items: 'required|array'
}
const validation = await Validator.validate(request.all(), rules)
```
Using integer works however I want to allow for decimals in the cost property.
Thanks in advance

**Comment by antwaanh** [comment]
@thetutlage @RomainLanz.
I'm confused about this as well. Any insight is greatly appreciated.

In my case I'm using a route validator.

App/Validators/StorePackage
```js
class StorePackage {
  get rules () {
    return {
      name: 'required|string|max:255',
      price: 'required|number',
    }
  }
}
```

Error
```json
[
    {
      "field": "price",
      "validation": "ENGINE_EXCEPTION",
      "message": "number is not defined as a validation"
    }
]
```

**Comment by thetutlage** [comment]
It is because you are reading the docs from indicative version 4 and Adonis uses the old version that is 3, docs for that version are here http://indicative-3.adonisjs.com/

Next release of AdonisJs will have to indicative v4.

**Comment by antwaanh** [comment]
@thetutlage thanks for the reply. 

Even [these docs](http://indicative-3.adonisjs.com/#indicative-schema-rules-number) have the number schema rule.

**Comment by RomainLanz** [comment]
Closing this since it's fixed in the new release of the framework (4.1).
Please upgrade (it takes only a few minutes).
